### PR TITLE
Clean up of readme doc

### PR DESCRIPTION
**Pull request by DonGloverUnity**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,131 +25,183 @@ At present time this open-source repository is maintained separately from the Ca
...
+maxPageCount                        | 10           | The maximum number of pages to allocate in the cache. The combination of `pageSize` and `maxPageCount` limits the overall memory footprint of the cache. When this threshold is reached, memory recovered using a Least Recently Used (LRU) algorithm.
+minFreeBlockSize                    | 1024         | The size of the minimum allocation unit, in bytes, within a page. You can specify a lower value for smaller projects.
+cachePath                           | `.cache_ram` | The path to the cache directory. Dirty memory pages are saved to disk periodically in this directory, and loaded at startup.
+persistence                         | true         | Enable saving and loading of page files to disk. If `false`, the cache is emptied during restart.
+persistenceOptions.autosave         | true         | When set to `true`, automatically save changed memory pages; set to `false` to disable. If `false`, pages are only saved when the cache server is stopped with the `q` console command or with upon a SIGTERM.
+persistenceOptions.autosaveInterval | 10000        | The frequency, in milliseconds, to save page that have changed.
```
Comment (stephen-palmer) [inline README.md]: page should be plural

File: README.md [hunk 2]
```diff
@@ -24,131 +25,183 @@ At present time this open-source repository is maintained separately from the Ca
...
+cleanupOptions.expireTimeSpan   | `P30D`      | [ASP.NET](https://msdn.microsoft.com/en-us/library/se73z7b9(v=vs.110).aspx) or [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601#Time_intervals) style timespan. Cache files that have not been accessed within this timespan are eligible for cleanup. For more information on duration syntax, see the [moment](https://momentjs.com/docs/#/durations/) library documentation.
+cleanupOptions.maxCacheSize     | 0           | The maximum size, in bytes, of the cache on disk. To bring the total disk utilization under this threshold, the cleanup script considers files for removal in least recently used order. Set the value to zero (0) to disable the cleanup feature.
+
+### Notes
+
+* cache_fs is backwards compatible with v5.x Cache Server directories.
+* Supports worker threads using the `--workers` option.
+* When you run the cleanup script, the value of the `expireTimeSpan` option is used to determine which file to delete files. If `maxCacheSize` is specified the script checks whether the cache exceeds the value of `maxCacheSize`. If it does, the script deletes files in least-recently-used order until the cache no longer exceed maxCacheSize.
```
Comment (stephen-palmer) [inline README.md]: "which file to delete files" doesn't make sense

File: README.md [hunk 3]
```diff
@@ -24,131 +25,183 @@ At present time this open-source repository is maintained separately from the Ca
 * [License](#license)
 
 ## Server Setup
-Download and install the latest LTS version of node from the [Node.JS website](`https://nodejs.org/en/download/`).
+
+Download and install the latest LTS version of Node.js from the [Node.JS website](https://nodejs.org/en/download/).
```
Comment (stephen-palmer) [inline README.md]: Feedback from customer .. confused about the specific version to download. Suggest adding v8.10.0 to this sentence.

File: README.md [hunk 4]
```diff
@@ -24,131 +25,183 @@ At present time this open-source repository is maintained separately from the Ca
...
-cachePath                           | `.cache_ram` | Path to cache directory. Dirty memory pages are saved to disk periodically in this directory, and loaded at startup.
-persistence                         | true         | Enable saving and loading of page files to disk. If `false`, the cache will be empty at every restart.
-persistenceOptions.autosave         | true         | `true` to periodically save dirty memory pages automatically; `false` to disable. If `false`, pages will only be saved when the cache server is stopped with the `q` console command or with SIGTERM.
-persistenceOptions.autosaveInterval | 10000        | Minimum interval in milliseconds to save dirty pages.
-
-#### Notes
-* Does not support worker threads
+pageSize                            | 100000000    | The page size, in bytes, used to grow the cache.
+maxPageCount                        | 10           | The maximum number of pages to allocate in the cache. The combination of `pageSize` and `maxPageCount` limits the overall memory footprint of the cache. When this threshold is reached, memory recovered using a Least Recently Used (LRU) algorithm.
```
Comment (stephen-palmer) [inline README.md]: memory _is_ recovered

File: README.md (hunk 1 above)
Comment (DonGloverUnity) [reply README.md]: Fixed.

File: README.md (hunk 2 above)
Comment (DonGloverUnity) [reply README.md]: Fixed.

File: README.md (hunk 3 above)
Comment (DonGloverUnity) [reply README.md]: Updated.

File: README.md (hunk 4 above)
Comment (DonGloverUnity) [reply README.md]: Fixed.

**Pull request merged**